doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1272 @@
+  * The :ref:`ug_bt_fast_pair_gatt_service_fmdn_info_callbacks_provisioning_state` section with changes to the FMDN API elements that are used for tracking of the FMDN provisioning state.
+
+Memfault integration
+--------------------
+
+* Added a new feature to automatically post coredumps to Memfault when network connectivity is available.
+  To enable this feature, set the :kconfig:option:`CONFIG_MEMFAULT_NCS_POST_COREDUMP_ON_NETWORK_CONNECTED` Kconfig option to ``y``.
+  Only supported for nRF91 Series devices.
+
+* Added a new feature to automatically capture and upload modem traces to Memfault with coredumps upon a crash.
+  To enable this feature, set the :kconfig:option:`CONFIG_MEMFAULT_NCS_POST_MODEM_TRACE_ON_COREDUMP` Kconfig option to ``y``.
+  Only supported for nRF91 Series devices.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``a2bc982b3379d51fefda3e17a6a067342dce1a8b``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Fixed an issue where an unusable secondary slot was cleared three times instead of once during cleanup.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``fdeb7350171279d4637c536fcceaad3fbb775392``, with some |NCS| specific additions.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline fdeb735017 ^beb733919d
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^fdeb735017
+
+The current |NCS| main branch is based on revision ``fdeb735017`` of Zephyr.
+
+.. note::
+   For possible breaking changes and changes between the latest Zephyr release and the current Zephyr version, refer to the :ref:`Zephyr release notes <zephyr_release_notes>`.
+
+Additions specific to |NCS|
+---------------------------
+
+|no_changes_yet_note|
+
+zcbor
+=====
+
+|no_changes_yet_note|
+
+Trusted Firmware-M
+==================
+
+|no_changes_yet_note|
+
+cJSON
+=====
+
+|no_changes_yet_note|
+
+Documentation
+=============
+
+* Added:
```

Comment:
Need an entry here for the LTE guide PR - https://github.com/nrfconnect/sdk-nrf/pull/21280, under Added:

 `* The :ref:`cellular_psm` page under the :ref:`ug_lte` documentation, and the documentation was spilt into subpages.